src/Type/Doctrine/ObjectMetadataResolver.php

```diff
@@ -60,11 +60,18 @@ public function getObjectManager(): ?ObjectManager
 
 	private function loadObjectManager(string $objectManagerLoader): ?ObjectManager
 	{
-		if (
-			!file_exists($objectManagerLoader)
-			|| !is_readable($objectManagerLoader)
-		) {
-			throw new \PHPStan\ShouldNotHappenException('Object manager could not be loaded');
+		if (!file_exists($objectManagerLoader)) {
```

Comment:
Please use is_file() here instead.

Reply:
I think behavior changes should be done separately. Here I mostly move code around without functional changes.

Reply:
I asked you to make a change. It will leave the code in a better state. It can be a separate commit.

Reply:
OK, done.